platform/javascript/os_javascript.cpp

```diff
@@ -708,7 +708,7 @@ Error OS_JavaScript::initialize(const VideoMode &p_desired, int p_video_driver,
 	video_driver_index = p_video_driver;
 
 	video_mode = p_desired;
-	// Can't fulfill fullscreen request during start-up due to browser security.
+	// Can't fulfil fullscreen request during start-up due to browser security.
```

Comment:
This one could be kept, `os_javascript.cpp` is our own. It's `platform/android/cpu-features.c` that must be reverted.

Reply:
2b29cff